src/app/[locale]/forgot-password/page.tsx

```diff
@@ -0,0 +1,94 @@
+'use client';
+
+import { Locale } from '@/locale-config';
+import { Channel } from '@/channel-config';
+import { useTranslations } from '@/core/server/useTranslations';
+import { useState } from 'react';
+import { requestPasswordReset } from '../(components)/serverSubmitHandlers';
+import { TextField } from '../(components)/textField';
+import Button from '@/app/daisyui/button';
+
+interface PageProps {
+  params: {
+    locale: Locale;
+    channel: Channel;
+  };
+}
+
+export default function RequestPasswordReset() {
+  const [email, setEmail] = useState<string>('');
+  const [dirty, setDirty] = useState<boolean>(false);
+  const [isSubmitted, setIsSubmitted] = useState<boolean>(false);
+  const t = useTranslations();
+
+  const handleEmailUpdated = (e: any) => {
+    setDirty(true);
+    setEmail(e.target.value);
+  };
+
+  let emailError = '';
+  if (dirty && !email) {
+    emailError = 'Please enter your email to continue.';
+  } else if (dirty && /\S+@\S+\.\S+/.test(email) === false) {
+    emailError = 'Please provide a valid email address.';
+  }
+
+  const handleSubmit = () => {
+    const formData = new FormData();
+    formData.append('email', email);
+    const respPromise = requestPasswordReset(formData);
+
+    respPromise.then((resp) => {
+      requestPasswordReset(formData);
+      setIsSubmitted(true);
+    });
+  };
+
+  if (isSubmitted) {
+    return <div className="py-7 justify-start items-start gap-2.5 inline-flex">
+      <div className="w-full md:w-1/2 p-10">
+        <div className="flex flex-col justify-start items-start w-full gap-6">
+          <h1 className="text-base-10 text-4xl font-bold ">
+            Check your inbox
```

Comment:
Can these strings be added to translations? It's okay if we have english-only for now. If you look at `dictionaries/en-us.json` you should see other similar strings that are embedded into the site.